stm32-modules/heater-shaker/firmware/motor_task/motor_policy.cpp

```diff
@@ -17,8 +18,10 @@ auto MotorPolicy::set_rpm(int16_t rpm) -> void {
     }
     int16_t current_speed = get_current_rpm();
     int16_t command_01hz = rpm * _01HZ / _RPM;
-    uint16_t ramp_time =
-        static_cast<int32_t>(std::abs(rpm - current_speed)) * RAMP_MS_PER_RPM;
+    uint16_t ramp_time = std::max(
```

Comment:
I see that you slightly changed the types here, but I'm still confused as to why it specifically needs to be unit16_t

Reply:
Because that's what the driver code's `ProgramSpeedRamp` function takes. That's pretty much it.